python/ray/rllib/ddpg/__init__.py

```diff
@@ -1,3 +1,8 @@
+from __future__ import absolute_import
```

Comment:
Hm looks like github didn't detect the moves correctly. Basically all of these files were moved unchanged (sans 2/1 renaming) from `ddpg` => `ddpg2` and vice versa.